pkgs/desktops/gnome/extensions/update-extensions.py

```diff
@@ -37,6 +36,10 @@
 for shell_version in supported_versions.keys():
     package_name_registry[shell_version] = {}
 
+dir_path = Path(__file__).resolve().parent
```

Comment:
This name is not really descriptive. Maybe something like `my_dir_path`, `updater_dir_path` or `script_location`.

Reply:
`current_dir_path`, `workdir_path`?

Reply:
That sounds too much to me like CWD/PWD.